react/__tests__/DocumentField.test.tsx

```diff
@@ -19,10 +21,10 @@ describe('DocumentField', () => {
 
     const input = getByLabelText(/document/i)
 
-    expect(input).toHaveValue('111.111.111-11')
+    await wait(() => void expect(input).toHaveValue('111.111.111-11'))
```

Comment:
Do you think it makes sense adding a test with a half-formatted cpf? like `061.00316744`

Reply:
you mean the user inputting `061.00316744` and see if after the blur it should be `061.003.167-44`?

Reply:
done in 0fff8b2

Reply:
Yes. Great.